### PR TITLE
PR: Change screenshot in read me spyder 4

### PR DESCRIPTION
**Pull request by juanis2112**
## Description of Changes

* [ ] Wrote at least one-line docstrings (for any new functions)
* [ ] Added unit test(s) covering the changes (if testable)

* [ ] Included a screenshot or animation (if affecting the UI, see [Licecap](https://www.cockos.com/licecap/))

 Change screenshot in read me spyder 4

### Issue(s) Resolved

Fixes #

### Affirmation

By submitting this Pull Request or typing my (user)name below,
I affirm the [Developer Certificate of Origin](https://developercertificate.org)
with respect to all commits and content included in this PR,
and understand I am releasing the same under Spyder's MIT (Expat) license.

I certify the above statement is true and correct:
Juanis2112

### CODE REVIEW

**Review by ccordoba12 — APPROVED** [review]
Thanks @juanis2112!

File: README.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ https://github.com/spyder-ide/spyder/graphs/contributors)*
 [![codecov](https://codecov.io/gh/spyder-ide/spyder/branch/master/graph/badge.svg)](https://codecov.io/gh/spyder-ide/spyder)
 [![Crowdin](https://badges.crowdin.net/spyder/localized.svg)](https://crowdin.com/project/spyder)
 
-![Screenshot of Spyder's main window](https://raw.githubusercontent.com/spyder-ide/spyder/master/img_src/screenshot.png)
+![Screenshot of Spyder's main window](https://raw.githubusercontent.com/spyder-ide/spyder/4.x/img_src/screenshot.png)
```
Comment (ccordoba12) [inline README.md:29]:
```suggestion
![Screenshot of Spyder's main window](https://raw.githubusercontent.com/spyder-ide/spyder/4.x/img_src/screenshot-v4.png)
```

**Review by ccordoba12 — CHANGES_REQUESTED** [review]
Since I have to merge this PR against `5.x` and `master`, please rename `img_src/screenshot.png` to `img_src/screenshot-v4.png`.

With that, the screenshots for Spyder 4 and 5 will be separate.

**Comment by dalthviz** [comment]
Closing as oudated

**Pull request closed without merging**